aiokafka/__init__.py

```diff
@@ -1,7 +1,5 @@
 __version__ = '0.0.1'
 
-from .client import AIOKafkaClient
-from .producer import SimpleAIOProducer, KeyedAIOProducer
-from .consumer import SimpleAIOConsumer
-
-(AIOKafkaClient, SimpleAIOProducer, KeyedAIOProducer, SimpleAIOConsumer)
+from .client import AIOKafkaClient  # noqa
```

Comment:
Please revert the change -- it should be done in different PR if any.
Honestly I doubt we need it at all.